packtools/sps/validation/funding_group.py

```diff
@@ -47,47 +47,62 @@ def validate_required_award_ids(self):
+                for item in ack.get("p") or []:
                     if item.get("look-like-award-id"):
-                        item["context"] = "fn[@fn-type='supported-by']"
+                        item["context"] = "ack"
                         errors.append(item)
-            if funding_statement_data := self.funding.funding_statement_data:
-                if funding_statement_data.get("look-like-award-id"):
-                    funding_statement_data["context"] = (
-                        "funding-group/funding-statement"
-                    )
-                    errors.append(funding_statement_data)
-
-            for error in errors:
-                yield build_response(
-                    title="Required funding-group/award-group with award-id and funding-source",
-                    parent=parent,
-                    item="award-group",
-                    sub_item="award-id",
-                    validation_type="exist",
-                    is_valid=False,
-                    expected="award-id and funding-source in award-group",
-                    obtained=None,
-                    advice="Found `{}` in `{}` ({}). Check it is a project contract. Add award-id ({}) in funding-group/award-group with respectives funding-source".format(
-                        error["look-like-award-id"],
-                        error["text"],
-                        error["context"],
-                        error["look-like-award-id"],
-                    ),
-                    data=error,
-                    error_level=self.params["error_level"],
+        if items := self.funding.financial_disclosure:
+            for item in items or []:
+                if item.get("look-like-award-id"):
+                    item["context"] = "fn[@fn-type='financial-disclosure']"
+                    errors.append(item)
+        if items := self.funding.supported_by:
+            for item in items or []:
+                if item.get("look-like-award-id"):
+                    item["context"] = "fn[@fn-type='supported-by']"
+                    errors.append(item)
+        if funding_statement_data := self.funding.funding_statement_data:
+            if funding_statement_data.get("look-like-award-id"):
+                funding_statement_data["context"] = (
+                    "funding-group/funding-statement"
                 )
+                errors.append(funding_statement_data)
+
+        for error in errors:
+            yield build_response(
+                title="Required funding-group/award-group with award-id and funding-source",
+                parent=parent,
+                item="award-group",
+                sub_item="award-id",
+                validation_type="exist",
+                is_valid=False,
+                expected="award-id and funding-source in award-group",
+                obtained=None,
+                advice="Found `{}` in `{}` ({}). Verify if this refers to a project contract. "
```

Comment:
@Rossi-Luciano troque `Verify if this refers to a project contract...` por `If {} is a project contract number, make  it with <award-id> and the corresponding financial sponsors with <funding-source> in <funding-group>. Consult the SPS documentation for more detail`

sendo `{}`= `error["look-like-award-id"]`